_sass/partials/_epub-base-typography.scss

```diff
@@ -57,7 +57,7 @@ $epub-base-typography: true !default;
     p {
         margin: 0;
         & + p {
-            text-indent: $line-height-default;
+            text-indent: $paragraph-indent;
```

Comment:
We're changing the default paragraph indent from `line-height-default` to `font-size-default` - just want to check that this was intended.

Reply:
Thanks, yes, this was deliberate. Over time and drawing from Karen's designs, I've noticed that for most projects the line height is a bit too much, and proportionally the font size makes for a more elegant indent.